src/cli/repo/info.rs

```diff
@@ -19,14 +22,24 @@ struct RepoInfo {
 pub struct Options {
     #[clap(long, env = "ENARX_CA_BUNDLE")]
     ca_bundle: Option<Utf8PathBuf>,
+    #[clap(long, default_value = "https://auth.profian.com/")]
+    oidc_domain: Url,
     #[clap(long, env = "ENARX_INSECURE_AUTH_TOKEN")]
     insecure_auth_token: Option<String>,
+    #[clap(long, env = "ENARX_CREDENTIAL_HELPER")]
+    credential_helper: Option<OsString>,
```

Comment:
These elements look like they could be refactored.

Reply:
Yes, it should be in #2021 , which does not belong in this PR.
Adding these was a massive pain...